README.md

```diff
@@ -33,6 +33,15 @@ case:
 $ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
 ```
 
+## Minimum Supported Rust Version (MSRV)
+
+At the moment we use test and support the following Rust compiler versions:
+
+* On Ubuntu we test with the latest stable compiler version, as accessible through `rustup`.
+* On Fedora we test with the compiler version included with the Fedora 33 release.
```

Comment:
Do we test with the _latest_ compiler version available in Fedora 33?

Reply:
I think we test with the one that comes with the OS release? CC @puiterwijk

Reply:
Yes, we test with the one that's included in Fedora 33.